lib/standard/runners/genignore.rb

```diff
@@ -0,0 +1,43 @@
+require "yaml"
+require_relative "rubocop"
+
+module Standard
+  module Runners
+    class Genignore
+      def call(config)
+        # Run Rubocop to generate the files with errors.
+        config.rubocop_options[:formatters] = [["files", "temp_exclude.txt"]]
+        config.rubocop_options[:format] = "files"
+        config.rubocop_options[:out] = "temp_exclude.txt"
+
+        File.delete("exclude.txt") if File.exist?("temp_exclude.txt")
```

Comment:
What's `exclude.txt`? It's only mentioned here.

As for `temp_exclude.txt`, I don't love spitting out a temporary file into a person's project directory, because if there's a failure it'll sit there and they won't know what it is. Maybe it'd be better to use Ruby's [tempfile](https://ruby-doc.org/stdlib-2.3.0/libdoc/tempfile/rdoc/Tempfile.html) instead?

Reply:
I've updated to the code to use Ruby's tempfile.  Thanks for the tip.